CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+v2.9.1
+======
+* Added conversion tests for `Calendar` format (`String` -> `Calendar`)
```

Comment:
As far as changes from this PR do not fix anything in production part of the library I think that there is no need to updating `CHANGELOG.md` - it could be confusing for the others.

Reply:
:+1: @gotfrydoh please revert the changelog update